static/js/zamboni/validator.js

```diff
@@ -343,11 +343,13 @@ function initValidator($doc) {
                     }
                 });
                 $context.append($code);
-            } else if (msg.line) {
+            } else if (msg.line && msg.column !== 'undefined') {
```

Comment:
Okay, I thought  you'd need typeof to make sure the string check works
or compare to undefined otherwise
 
 
On Thu, May 26, 2016, at 09:48 PM, Andy McKay wrote:
> In static/js/zamboni/validator.js[1]:


> > @@ -343,11 +343,13 @@ function initValidator($doc) { >
> > } >                  }); >                  $context.append($code);
> > > -            } else if (msg.line) { > +            } else if
> > (msg.line && msg.column !== 'undefined') {
>
> All the other checks for undefined in the source are string checks
> like this.
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub[2]
>
 

Links:

  1. https://github.com/mozilla/addons-server/pull/2777#discussion_r64808953
  2. https://github.com/mozilla/addons-server/pull/2777/files/dd38d9a1956303ed5f3935b2702c4a1f85e64398#r64808953